Modules/Core/Transform/include/itkEuler3DTransform.hxx

```diff
@@ -135,6 +135,12 @@ template <typename TParametersValueType>
 void
 Euler3DTransform<TParametersValueType>::SetFixedParameters(const FixedParametersType & parameters)
 {
+  if (parameters.size() < InputSpaceDimension)
```

Comment:
Should this be "  != " rather than " < " ? 

along with a change to the comment below.

Reply:
Please see line 147/153. This transform supports legacy transform with a missing parameter. 

Additionally, the "<" only checks what is required here. A derived transform could support more fixed parameters.

Reply:
> Should this be " != " rather than " < " ?

@hjmjohnson Good question, thanks Hans. I do have an answer, though  😃  First of all, the aim of the commit is to fix a crash. At https://github.com/InsightSoftwareConsortium/ITK/blob/v5.1.2/Modules/Core/Transform/include/itkEuler3DTransform.hxx#L140-L142

    for (unsigned int i = 0; i < InputSpaceDimension; i++)
    {
      c[i] = this->m_FixedParameters[i] = parameters[i];  // Crashes when m_FixedParameters.size() is less than three.
    }

Such a crash _only_ occurs for m_FixedParameters.size() < 3.

Secondly, `m_FixedParameters.size()` may be three or four. Both sizes are supported. I can imagine that in the future, more, larger sizes will be supported. (Extra, optional fixed parameters.) But less than three is always an error!  OK?

Reply:
Sorry @blowekamp I just see you already answered the question from Hans.

Reply:
@blowekamp There appears to be a strong agreement between the two answers of us  😄

Reply:
Thank you for the explanation.